samples/databus/custom-serializer/Core_6/Shared/MessageWithLargePayload.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using NServiceBus;
+
+//the data bus is allowed to clean up transmitted properties older than the TTBR
+[TimeToBeReceived("00:01:00")]
```

Comment:
TBH I have no idea if it is required but that would be odd given there is no automatic cleanup ;-)